packages/atoms/src/Modal/SlideDownModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from 'react';
+import { View } from 'react-native';
+import { Modal } from './Modal';
+
+type ModalProps = {
+    isModalVisible: boolean;
+    setIsModalVisible: (isVisible: boolean) => void;
```

Comment:
I prefer not using setter of state directly as props for components.  How about something like **onVisibilityChange**?

Reply:
[aaaef34](https://github.com/trezor/trezor-suite/pull/5493/commits/aaaef34b96d5d160c4f32566e829f4234266d96e)